src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "AvenirBold";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+  --main-text-color: #3d4e61;
```

Comment:
CSS variables are usually set in the `:root` block at the beginning of the file.